.gitignore

```diff
@@ -15,5 +15,3 @@ log
 
 # rspec failure tracking
 .rspec_status
-
-*.gemfile.lock
```

Comment:
Not sure about this change. If we don't want to got this way we will have to update rubocop.